src/core/discontinuity.ts

```diff
@@ -7,10 +7,10 @@ import EventEmitter from './utils/event-emitter.js';
  */
 export interface HandlesDiscontinuity {
   /**
-   * A promise of the channel that this object is associated with. The promise
+   * A channel that this object is associated with. The promise
```

Comment:
Needs update to remove the old promise description.

Reply:
Not sure how I missed that...